src/connectors/numeric-refinement-list/connectNumericRefinementList.js

```diff
@@ -77,49 +74,32 @@ export default function connectNumericRefinementList(renderFn) {
         };
 
         this._createURL = state => facetValue => createURL(refine(state, attributeName, options, facetValue));
-
-        const items = options.map(facetValue =>
-          ({
-            ...facetValue,
-            isRefined: isRefined(helper.state, attributeName, facetValue),
-            attributeName,
-          })
-        );
+        this._prepareItems = state => options.map(({start, end, name: label}) => ({
+          label,
+          value: window.encodeURI(JSON.stringify({start, end})),
```

Comment:
We should be able to pass this value into html strings, and the user doesn't need to know about the structure of it. So I guess that's fine.